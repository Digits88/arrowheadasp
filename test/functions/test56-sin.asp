<!-- Test Sin function.
-->

Sin(0) = <%= Sin(0) %><br>
Sin(3.1415926) = <%= Sin(3.1415926) %><br>
Sin(-3.1415926) = <%= Sin(-3.1415926) %><br>
