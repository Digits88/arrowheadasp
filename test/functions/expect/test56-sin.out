<!-- Test Sin function.
-->

Sin(0) = 0<br>
Sin(3.1415926) = 5.358979317^[0-9]*$E-^0?$8<br>
Sin(-3.1415926) = -5.35897931^[0-9]*$E-^0?$8<br>
